.github/workflows/build.yml

```diff
@@ -12,7 +12,12 @@ jobs:
       fail-fast: false
       matrix:
         platform: [ubuntu-18.04]
-        os_name: [linux_openresty, linux_tengine, linux_apisix_master_luarocks, linux_apisix_current_luarocks, linux_openresty_mtls]
+        os_name:
```

Comment:
Then we may need to update the platform field, too.

Reply:
+1